src/core/webrtc/simple-room.ts

```diff
@@ -475,16 +478,15 @@ export const getRoom = (id: string) => {
         room.livekitRoom?.off(RoomEvent.ActiveSpeakersChanged, fn)
       }
     },
-    sendData: (data, recipientIds) => {
+    sendData: (data, recipientIds = []) => {
       const encoded = encoder.encode(JSON.stringify(data))
       const participants = recipientIds
-        ?.map((x) => room.livekitRoom?.getParticipantByIdentity(x)?.sid)
+        ?.map((x) => room.livekitRoom?.getParticipantByIdentity(x))
         .filter(Boolean)
-      return localParticipant.publishData(
-        encoded,
-        DataPacket_Kind.RELIABLE,
-        participants,
-      )
+      return localParticipant.publishData(encoded, {
+        reliable: true,
+        destinationIdentities: participants.map((x) => x.identity),
+      })
```

Comment:
does this requires changes on studio-next-client where `room.onData` is used?

Reply:
It does not seem to require any changes on the receiving end